app/views/spree/shared/_bottom_footer.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="footer-copyright">
+   <div class="row no-gutters">
+      <div class="col-xl-2 tablet-hide"></div>
+      <div class="col-xl-4 col-lg-6">
+         <p>Copyright &copy; 1990-2021 Natures Flavors, Inc. All rights reserved.</p>
```

Comment:
Add this copyright information into the partial and call the partial here

<%#= render 'spree/shared/copyright' %>